cividiscount.php

```diff
@@ -437,10 +437,13 @@ function cividiscount_civicrm_buildAmount($pageType, &$form, &$amounts) {
           }
         }
       }
+      if ($autodiscount) {
+        break;
+      }
```

Comment:
I'm confused about this line. It seems to be breaking a `foreach` loop that is already ending 1 line later anyway. Isn't the closing `}` one line down from this the end of the loop?

Reply:
Not sure if I understand your question. Are you asking why we're breaking out?  The foreach cycles through all discounts.  

It _could_ end one line later if we break out, otherwise it will always end on the highest discount ID we have.

Reply:
My mistake, I was confusing `break` with `continue`. Need coffee.

Reply:
haha, no worries!